crates/rpc/rpc-types/src/net.rs

```diff
@@ -126,6 +128,9 @@ pub enum NodeRecordParseError {
     /// Invalid discport
     #[error("Failed to discport query: {0}")]
     Discport(ParseIntError),
+    /// Conversion from type [`Enr<SecretKey>`] failed.
+    #[error("failed to convert enr into node record")]
+    ConversionFromEnrFailed,
```

Comment:
this is more like `MissingIpInfo`?

we could unify this with this check here, I think.

https://github.com/paradigmxyz/reth/blob/4d798c7b655255b84cbbcd3af2e8827de81e105e/crates/rpc/rpc-types/src/net.rs#L148-L148